torch/utils/cpp_extension.py

```diff
@@ -276,8 +324,8 @@ def build_extensions(self):
             self._define_torch_extension_name(extension)
             self._add_gnu_cpp_abi_flag(extension)
 
-        # Register .cu and .cuh as valid source extensions.
-        self.compiler.src_extensions += ['.cu', '.cuh']
+        # Register .cu, .cuh and .hip as valid source extensions.
+        self.compiler.src_extensions += ['.cu', '.cuh', '.hip']
```

Comment:
Are .cu/.cuh files valid for ROCm?

Reply:
Yes. `hipify_python.hipify` (called here: https://github.com/pytorch/pytorch/pull/32669/files#diff-c87f1d9ef353fab9c4de839fe5adc8eeR33 ) will translate those from CUDA to HIP, to be consumed by the ROCm compiler.